README.md

```diff
@@ -3,3 +3,58 @@
 This repository contains the AML Spec which can be viewed either in [HTML](http://a.ml/aml-spec/vocabularies/) or Markdown format:
 - [AML Vocabularies 1.0](vocabularies.md)
 - [AML Dialects 1.0](dialects.md)
+
+## How to run locally
```

Comment:
I would move this section to CONTRIBUTING so as to not clutter the README since no one needs this section other than prospective contributors.